src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -749,3 +749,10 @@ COOP_PINVOKE_HELPER(void, RhGetCurrentThreadStackBounds, (PTR_VOID * ppStackLow,
 {
     ThreadStore::GetCurrentThread()->GetStackBounds(ppStackLow, ppStackHigh);
 }
+
+ThreadExitCallback g_threadExitCallback;
```

Comment:
Do we want to support this callback on Windows (of course, we would need to add a thread_local as well)?